Update install instructions (#14)
BanDev/Lux: README.md

<p>Lux is a command-line interface for controlling and monitoring Govee lighting strips built in Go. Lux provides it's users with the ability to manage their lighting strips from their desktop computer from any network with an internet connection.</p>

<h2 id="installation">Installation</h2>

> <p>You can install Lux however you wish, we provide a simple installer executable for each release but if that's not your style you can easily compile Lux yourself directly from source. </p>

Reviewer: ```suggestion
<p>We provide a simple installer for each release for Windows but if that's not suitable for you, you can easily compile Lux directly from source. </p>
```

> <p>You can attain the latest copy of the Lux installer from our <a href="https://github.com/BanDev/Lux/releases">releases</a> page on GitHub, or through <a href="https://docs.microsoft.com/en-us/windows/package-manager/winget/">Winget</a> using the command below:</p>

Reviewer: ```suggestion
<p>You can get the latest copy of Lux from our <a href="https://github.com/BanDev/Lux/releases">GitHub releases</a>, or via <a href="https://docs.microsoft.com/en-us/windows/package-manager/winget/">Winget</a></p>
```

> <pre>

Reviewer: ```suggestion
```sh
```

winget install BanDev.Lux

> </pre>

Reviewer: ```suggestion
```
```

> <blockquote>If you chose to self compile, you need to ensure you add the program's `lux.exe` path to your `Path` environment variables in order for Lux to be detected as a command in PowerShell & CMD.</blockquote>

Reviewer: ```suggestion
<blockquote>If you choose to self compile, you will need to add the path to `lux.exe` to your `Path` environment variables in order for Lux to be detected as a command in PowerShell & CMD.</blockquote>
```

> <p>If you're already using a terminal, restart it to update environment variables and then run the following to begin setting up your instance of Lux.</p>

Reviewer: ```suggestion
<p>If you're already using a terminal, restart it so that it can register the changes to the environment variables. Then, run the following command to begin setting up your instance of Lux.</p>
```
